### ISSUE TITLE
help!

### ISSUE THREAD

**Issue by Iso1558** [CocoaPods/CocoaPods#7763]
ArgumentError - Malformed version number string
/Users/tami/.rvm/rubies/ruby-2.4.1/lib/ruby/site_ruby/2.4.0/rubygems/version.rb:209:in `initialize'
/Users/tami/.rvm/rubies/ruby-2.4.1/lib/ruby/site_ruby/2.4.0/rubygems/version.rb:200:in `new'
/Users/tami/.rvm/rubies/ruby-2.4.1/lib/ruby/site_ruby/2.4.0/rubygems/version.rb:200:in `new'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb:123:in `embedded_content_settings'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb:68:in `generate'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb:39:in `save_as'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb:112:in `update_changed_file'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb:96:in `block in create_xcconfig_file'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb:93:in `each'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb:93:in `create_xcconfig_file'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb:18:in `block in install!'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/user_interface.rb:142:in `message'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb:14:in `install!'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator.rb:181:in `block (2 levels) in install_libraries'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator.rb:179:in `each'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator.rb:179:in `block in install_libraries'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/user_interface.rb:142:in `message'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator.rb:167:in `install_libraries'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer/xcode/pods_project_generator.rb:65:in `generate!'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:184:in `block in generate_pods_project'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/user_interface.rb:64:in `section'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:183:in `generate_pods_project'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/installer.rb:119:in `install!'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/command/install.rb:41:in `run'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/lib/cocoapods/command.rb:52:in `run'
/Users/tami/.rvm/gems/ruby-2.4.1@global/gems/cocoapods-1.5.2/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
/Users/tami/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
/Users/tami/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.

Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=Malformed+version+number+string+&type=Issues

If none exists, create a ticket, with the template displayed above, on:
https://github.com/CocoaPods/CocoaPods/issues/new

Be sure to first read the contributing guide for details on how to properly submit a ticket:
https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md

Don't forget to anonymize any private data!

Looking for related issues on cocoapods/cocoapods...
 - ArgumentError - Malformed version number string $(inherited)
   https://github.com/CocoaPods/CocoaPods/issues/6634 [closed] [5 comments]
   a minute ago

 - [!] Oh no, an error occurred.
   https://github.com/CocoaPods/CocoaPods/issues/7756 [closed] [5 comments]
   3 days ago

 - Concrete error for loading podspecs via git should be shown
   https://github.com/CocoaPods/CocoaPods/issues/5507 [closed] [12 comments]
   12 Jul 2017

and 27 more at:
https://github.com/cocoapods/cocoapods/search?q=Malformed%20version%20number%20string%20&type=Issues&utf8=✓

**Comment by zedixns** [comment]
Hi there, I don't think you have enough details in order for people to assist you.
Please follow the template that you have when summiting a new issue in order to help the CocoaPods team figure out the problem faster.

What I can tell you is that I am experiencing a similar issue #7765

**Comment by DanToml** [comment]
Closing this in favour of #7765

**Comment by segiddins** [comment]
This should now be fixed in CocoaPods 1.5.3!
